Automattic/pocket-casts-ios
# Fixed close button not rendering properly when using Dark Contrast theme

Fixes #552 (introduced by commit https://github.com/Automattic/pocket-casts-ios/commit/42aa2754814640957063be31c9b5faadc56da509)

The referenced commit added an SVG image named `close`, but `PlayerImages.xcassets` already contains a PNG image asset with the same name, which results in `ModalCloseButton` loading the incorrect image and thus resulting in the rendering bug. This PR renames the newly added SVG (and its references) to resolve the conflict.

## To test
- Select the Dark Contrast theme
- Open Settings->About
- The close button should be rendered as a white circle with a black x. Before the change, the button was rendered as a solid white circle. 

## Checklist
